deploy.sh

```diff
@@ -46,6 +46,10 @@ git commit -m "Travis CI Clean Deploy : ${SHA}"
 git checkout $SOURCE_BRANCH
 
 # Actual building and setup of current push or PR.
+npm cache clean --force
+npm install npm@latest -g
+rm package-lock.json
+npm install -g @angular/cli
```

Comment:
Why don't we have the things already installed? I think they can go to some Travis hook.

Reply:
I am also not sure about it. But I had to add it because ```ng command was not found```. I am not sure but we can use this command below ```yarn install --ignore-engines``` in ```install``` hook in travis. But we can test firstly the give changes should work.

Reply:
Okay.